test/ctia/store_test.clj

```diff
@@ -0,0 +1,27 @@
+(ns ctia.store-test
+  (:require [ctia.store :as sut]
+            [ctia.test-helpers.http :refer [app->APIHandlerServices]]
+            [ctia.test-helpers.fixtures :as fixt]
+            [ctia.test-helpers.core :as helpers :refer [GET POST PATCH]]
+            [ctim.examples.incidents :refer [incident-minimal]]
+            [clojure.test :refer [deftest testing is]]))
+
+(deftest list-all-pages
+  (testing "list all pages shall properly list all results for given entity and parameters"
+    (helpers/fixture-ctia-with-app
+     (fn [app]
+       (let [services (app->APIHandlerServices app)
+             incidents (->> (fixt/n-doc incident-minimal 10)
+                            (map #(assoc % :source "list-all-pages")))
+             bulk-result (helpers/POST-bulk app {:incidents incidents})]
+         (clojure.pprint/pprint services)
+         (clojure.pprint/pprint (sut/list-all-pages :incident
```

Comment:
this call fails for now because the services initialization is different in test and the `read-store` fonction seem false:

In CTIA server:
```
 :StoreService
 {:read-store
#object[ctia.store_service$eval70980$store_service_fn__GT_varargs__70985$fn__70986$fn__70987 0x2d4d098 "ctia.store_service$eval70980$store_service_fn__GT_varargs__70985$fn__70986$fn__70987@2d4d098"],
  :write-store  #object[ctia.store_service$eval70980$store_service_fn__GT_varargs__70985$fn__70986$fn__70987 0x710d3549 "ctia.store_service$eval70980$store_service_fn__GT_varargs__70985$fn__70986$fn__70987@710d3549"]},
```

In Tests:
```
 :StoreService
 {:read-store
  #object[clojure.core$partial$fn__5839 0x20ff18fb "clojure.core$partial$fn__5839@20ff18fb"],
  :write-store
  #object[clojure.core$partial$fn__5839 0x12306629 "clojure.core$partial$fn__5839@12306629"]},
```